clang/unittests/Format/TokenAnnotatorTest.cpp

```diff
@@ -3286,6 +3286,14 @@ TEST_F(TokenAnnotatorTest, BlockLBrace) {
   EXPECT_BRACE_KIND(Tokens[4], BK_Block);
   EXPECT_TOKEN(Tokens[5], tok::l_brace, TT_BlockLBrace);
   EXPECT_BRACE_KIND(Tokens[5], BK_Block);
+
+  Tokens = annotate("[foo bar:{{0, 1}}];", getLLVMStyle(FormatStyle::LK_ObjC));
```

Comment:
i believe this case was already WAI (somehow), can you add another selector after `}}`. e.g: `[foo bar:{{0, 1}} baz: baz];`